controllers/flowlogspipeline/flp_reconciler.go

```diff
@@ -123,8 +126,8 @@ func (r *FLPReconciler) GetServiceName(kafka *flowsv1alpha1.FlowCollectorKafka)
 }
 
 func (r *FLPReconciler) Reconcile(ctx context.Context, desired *flowsv1alpha1.FlowCollector) error {
-	for _, singleFlp := range r.singleReconcilers {
-		err := singleFlp.Reconcile(ctx, desired)
+	for i := 0; i < len(r.singleReconcilers); i++ {
```

Comment:
Maybe it's more "idiomatic":
```go
for i := range r.singleReconcilers
```
but do it as you prefer.

Reply:
Yes, thanks